src/main/java/org/selenium/pages/MyAccountPage.java

```diff
@@ -0,0 +1,32 @@
+package org.selenium.pages;
+
+import org.openqa.selenium.By;
+
+import java.util.Locale;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class MyAccountPage extends BasePage{ ;
+    private By waitContainer = By.xpath("//div[@id='columns']");
+    private By title = By.xpath("//h1[@class='page-heading']");
+    private By buttonWish = By.xpath("//a[@title='My wishlists']");
+
+    public String getTitleExpected() {
```

Comment:
The method name shouldn't have expected/actual in the name. The method itself only returns value. You should decide on the test level what is actual/expected
More suitable name is  getTitle() or getTitleText()